component-studio-integration/src/main/java/org/talend/sdk/component/studio/ComponentModel.java

```diff
@@ -238,7 +238,7 @@ public String getVersion() {
     @Override // TODO This is dummy implementation. Correct impl should be added soon
     public List<? extends IElementParameter> createElementParameters(final INode node) {
         ElementParameterCreator creator = new ElementParameterCreator(this, detail, node);
-        List<IElementParameter> parameters = (List<IElementParameter>) creator.createParameters();
+        List<IElementParameter> parameters = (List<IElementParameter>) creator.createParameters(node);
```

Comment:
seems, node parameter is not required here. It is passed to ElementParameterCreator constructor.
Thus, you don't need to pass it to ElementParameterCreator.addComponentParameters() method. You may use the field directly

Reply:
That's reasonable! thanks! I missed that. I'll fix it.